gst/tensor_decoder/tensordec.c

```diff
@@ -468,6 +509,13 @@ gst_tensordec_set_property (GObject * object, guint prop_id,
     case PROP_MODE:
       self->mode = g_value_dup_string (value);
       break;
+    case PROP_MODE_OPTION1:
+      if (g_strcmp0 (self->mode, "image_labeling") == 0) {
```

Comment:
In the case of using strcmp, some static code analysis tool may give you a guide to use str'n'cmp rather than strcmp. I am not sure that it is ok with g_strcmp0. You need to check it out.

Reply:
strncmp is looks safe, but I just try to use glib function. if there is problem using g_strcmp0, I will change to strncmp as you recommand

Reply:
Let's fix it (change to strncmp) if SVACE/Prevent complains or gstreamer maintainers complain. Until then, let's keep using g_strcmp0.